### PR TITLE
[Kibana] Update kibana service template

### PR DESCRIPTION
**Pull request by natebwangsut**
- Added an option to add annotations(s) to service resource.

Signed-off-by: natebwangsut <nate.bwangsut@gmail.com>

- [x] Chart version *not* bumped (the versions are all bumped and released at the same time)
- [x] README.md updated with any new values or changes
- [x] Updated template tests in `${CHART}/tests/*.py` 
- [ ] Updated integration tests in `${CHART}/examples/*/test/goss.yaml`

### CODE REVIEW

File: kibana/README.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ helm install --name kibana elastic/kibana --version 7.1.0 --set imageTag=7.1.0
...
 | `ingress`                 | Configurable [ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) to expose the Kibana service. See [`values.yaml`](./values.yaml) for an example                                                                                                                                                                                       | `enabled: false`                                                                                                          |
+| `service`                 | Configurable [service](https://kubernetes.io/docs/concepts/services-networking/service/) to expose the Kibana service. See [`values.yaml`](./values.yaml) for an example                                                                                                                                                                                       | `enabled: false`                                                                                                          |
```
Comment (Crazybus) [inline kibana/README.md]: I think the `enabled: false` is a copy paste error from the ingress definition above.

```
type: ClusterIP
port: 5601
annotations: {}
```

Should be the defaults

Comment (natebwangsut) [reply kibana/README.md]: I've fixed the defaults on README.md.

Comment (natebwangsut) [reply kibana/README.md]: Also, I've added

- Default empty annotation test case
- Test cases where the value has been added into annotations tag.

Comment (natebwangsut) [reply kibana/README.md]: I'll resolve the comments here as the changes have been made.

**Comment by natebwangsut** [comment]
Any update ?

**Comment by Crazybus** [comment]
jenkins test this please

File: kibana/tests/kibana_test.py [hunk 2]
```diff
@@ -41,6 +41,9 @@ def test_defaults():
 
     assert r['deployment'][name]['spec']['strategy']['type'] == 'Recreate'
 
+    # Make sure that the default 'annotation' dictionary is empty
+    assert not r['service'][name]['metadata']['annotations']
```
Comment (Crazybus) [inline kibana/tests/kibana_test.py]: This one is failing in CI with:

```
08:23:59 >       assert not r['service'][name]['metadata']['annotations']
08:23:59 E       KeyError: 'annotations'
```

I think you want:

```
assert 'annotations' not in r['service'][name]['metadata']
```

Comment (natebwangsut) [reply kibana/tests/kibana_test.py]: Alright, let me fix this real quick

**Comment by Crazybus** [comment]
jenkins test this please

**Pull request merged**